tests/reward_forwarder/test_deposit_reward.py

```diff
@@ -0,0 +1,101 @@
+import pytest
+import brownie
+
+WEEK = 86400 * 7
+
+
+def test_reward_deposit(alice, bob, charlie, child_gauge, reward_forwarder, reward_token):
+
+    reward_token._mint_for_testing(charlie, 10 ** 26, {"from": alice})
+    reward_forwarder.allow(reward_token, {"from": alice})
+
+    child_gauge.set_manager(bob, {"from": alice})
+    child_gauge.add_reward(reward_token, reward_forwarder, {"from": bob})
+
+    reward_token.transfer(reward_forwarder, 10 ** 20, {"from": charlie})
+    reward_forwarder.deposit_reward_token(reward_token, {"from": charlie})
+
+    assert reward_token.balanceOf(child_gauge) == 10 ** 20
+
+
+def test_reward_token_approval(alice, bob, charlie, child_gauge, reward_forwarder, reward_token):
```

Comment:
This test name should be a little more clarifying. This test verifies that `deposit_reward_token` reverts if a previous `allow` call wasn't made.

Reply:
Something like `test_reward_deposit_reverts_without_allowance`